Plugins/Flow.Launcher.Plugin.PluginsManager/Main.cs

```diff
@@ -3,20 +3,28 @@
 using Flow.Launcher.Plugin.PluginsManager.ViewModels;
 using Flow.Launcher.Plugin.PluginsManager.Views;
 using System.Collections.Generic;
+using System.Linq;
 using System.Windows.Controls;
+using Flow.Launcher.Infrastructure;
+using System;
+using System.Threading.Tasks;
 
 namespace Flow.Launcher.Plugin.PluginsManager
 {
     public class Main : ISettingProvider, IPlugin, ISavable, IContextMenu, IPluginI18n
     {
-        internal PluginInitContext Context { get; set; }
+        internal static PluginInitContext Context { get; set; }
```

Comment:
does this need to be static, since we are passing it through constructors already

Reply:
Oh we shouldn't make it static, though I think it is singleton. That has been changed to because I want to pass Context.API to the delegate to the default hotkey list.

Reply:
Let me check if there is a better way of passing variable.